code/go/0chain.net/smartcontract/zcnsc/authorizer_test.go

```diff
@@ -243,7 +243,7 @@ func Test_AuthorizerHealthCheck(t *testing.T) {
 
 	tr.ClientID = globalNode.ZCNSConfig.OwnerId
 
-	addAuthorizerPayload := CreateAuthorizerParam(tr.ClientID, tr.PublicKey)
+	addAuthorizerPayload := CreateAuthorizerParam("random_authorizer_delegate_wallet", tr.PublicKey)
```

Comment:
Please add a constant for the string both the places

Reply:
It's fine. We just need a random string. It is not utilised in test.